events/FlavorTUR.txt

```diff
@@ -933,13 +933,14 @@ country_event = {
 		NOT = {
 			has_global_flag = wih_kosem_tur_flag
 		}
+		owns = 161
```

Comment:
neit said "requires Naxos to be ruled by TUR or its non trib"
check on https://eu4.paradoxwikis.com/Triggers, theres one that does exactly this